genie-agent/src/main/java/com/netflix/genie/agent/cli/JobRequestArgumentsImpl.java

```diff
@@ -64,10 +64,10 @@
     private boolean interactive;
 
     @Parameter(
-        names = {"--archivalDisabled"},
-        description = "Disable server-side archival of logs (does not disable logs upload and short-term retrieval)"
+        names = {"--archiveLocationPrefix"},
+        description = "Prefix for job folder archival location(Currently only s3 URIs are supported)"
     )
-    private boolean archivalDisabled;
+    private String archiveLocationPrefix;
```

Comment:
Consider adding a validator and a converter to this option.
Validator rejects invalid values during parsing, rather than getting surprised by a bad URI at runtime.
Converter can be used to make this parameter a URI or whatever else makes sense. Rather than converting to URI every time you use it.

Reply:
Sure